src/main/user-impl/java/com/mysql/cj/jdbc/ha/ca/plugins/NodeMonitoringFailoverPlugin.java

```diff
@@ -99,6 +100,7 @@ public NodeMonitoringFailoverPlugin(
     this.propertySet = propertySet;
```

Comment:
next is often a keyword reserved to iterators - can we rename the next variable to something more meaningful? If it's a plugin, it can be nextPlugin or something...